XamarinCommunityToolkit/Views/CameraView/CameraViewRenderer.ios.cs

```diff
@@ -158,7 +198,8 @@ async void HandleShutter(object sender, EventArgs e)
 			{
 				case CameraCaptureOptions.Default:
 				case CameraCaptureOptions.Photo:
-					await Control?.TakePhoto();
+					if (Control != null)
+						await Control.TakePhoto();
```

Comment:
Is there any particular reason to do this?

Reply:
Previously it was
`await Control?.TakePhoto();`
If Control is null, it would throw an exception because you can't await null, only Tasks.

Reply:
BTW should we make all awaits "configure await - FALSE" ?

Reply:
I would not say all... Only if the rest of the method should not be run on the captured thread (e.g. the UI thread).

Reply:
Oh interesting! I would say it knows what to do with that null. Thanks!

Yeah, the ConfigureAwait is something we should probably more aware of as well.